modules/axhal/src/irq.rs

```diff
@@ -33,3 +34,11 @@ pub(crate) fn register_handler_common(irq_num: usize, handler: IrqHandler) -> bo
     warn!("register handler for IRQ {} failed", irq_num);
     false
 }
+
+#[reg_trap_handler(IRQ)]
```

Comment:
no `#[reg_trap_handler(PAGE_FAULT)]` currently ,right?

Reply:
Not required for the unikernels.